src/servers/ZoneServer2016/managers/fairplaymanager.ts

```diff
@@ -582,6 +585,77 @@ export class FairPlayManager {
     return true;
   }
 
+  checkAimVector(server: ZoneServer2016, client: Client, orientation: number) {
+    if (!this.useFairPlay || client.character.weaponStance != 2) return;
+    for (const a in server._characters) {
```

Comment:
maybe we should  only loop on the characters spawned client side and not all ones ?